misc/prerequisites.sh

```diff
@@ -302,6 +305,24 @@ install_ffmpeg()
         #ADDI_EXTRA_LIBS="-lpthread"
     fi
     
+    # If there is an enable-niquadra option, add patch from libxcoder_quadra-path
+    if [ "$NETINT_QUADRA_HWACCELS" = true ] ; then
+        echo "we are applying the patch founded in $NETINT_QUADRA_PATCH_PATH"
+        patch_name=$(basename $NETINT_QUADRA_PATCH_PATH)
+        cp $NETINT_QUADRA_PATCH_PATH ${DIR}
+        cd ${DIR} && patch -t -p 1 < $patch_name
+        if [ "$NETINT_QUADRA_XCODER_COMPILE_PATH" != "" ] ; then
+            cd $NETINT_QUADRA_XCODER_COMPILE_PATH && bash build.sh && ldconfig #the compilation of libxcoder can be done before
+        fi
+        ADDI_LIBS+=" --enable-ni_quadra --enable-avfilter  --enable-pthreads "
+        ADDI_ENCODER+=",h264_ni_quadra,h265_ni_quadra"
```

Comment:
What are the license implications of this? Is releasing a pre-built docker container with this compliant wit AGPL?

Reply:
I'm not sure about licensing and how this works for AGPL. wouldn't this be the same as all the others? Nvidia driver is proprietary. so is the logan and quadra.  OME uses it through ffmpeg which is compiled with nvidia and netint support. i do believe the one header file that ffmpeg uses for nvidia is open source. but everything else is close source. I'm not affiliated with netint. so i'm not sure

I just assumed since you already had logan support that quadra would be fine. this section is basically just a copy of how logan's prerequisites work..